--- a/third_party/renderdoc/renderdoc_app.h
+++ b/third_party/renderdoc/renderdoc_app.h
@@ -29,11 +29,13 @@
 #endif
 
 #if defined(WIN32)
-	#define RENDERDOC_CC __cdecl
+    #define RENDERDOC_CC __cdecl
 #elif defined(__linux__)
-	#define RENDERDOC_CC
+    #define RENDERDOC_CC
+#elif defined(__APPLE__)
+    #define RENDERDOC_CC
 #else
-	#error "Unknown platform"
+    #error "Unknown platform"
 #endif
 
 #ifdef __cplusplus
